Assets/HoloToolkit-Preview/QRTracker/Scripts/SpatialGraphCoordinateSystem.cs

```diff
@@ -1,66 +1,70 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
+
 #if WINDOWS_UWP
 using Windows.Perception.Spatial;
 #endif
+
 namespace HoloToolkit.Unity
 {
     public class SpatialGraphCoordinateSystem : MonoBehaviour
     {
 #if WINDOWS_UWP
```

Comment:
Here's one of those cases were we're not being consistent with the compiler define symbols. Will this affect the project?

Reply:
These mean two different things. One is wrapping a Windows platform call, which will only run under `#if WINDOWS_UWP`. The other times, I'm either wrapping Unity's platform-specific APIs or references to a plugin that only exists in UWP/editor.

Reply:
No worries, I just want to make sure we're using these correctly. Only difference from docs is that it's also targeting .NET Scripting Backend.

[Unity Docs](https://docs.unity3d.com/Manual/PlatformDependentCompilation.html)

UNITY_WSA | #define directive for Universal Windows Platform. Additionally, NETFX_CORE is defined when compiling C# files against .NET Core and using .NET scripting backend.
-- | --
UNITY_WSA_10_0 | #define directive for Universal Windows Platform. Additionally WINDOWS_UWP is defined when compiling C# files against .NET Core.

Reply:
That description is slightly misleading, as that's not the difference: https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/2017.4.2.0/CodingGuidelines.md#use-platform-dependent-compilation-to-ensure-the-toolkit-wont-break-the-build-on-another-platform

Reply:
That's exactly what I was looking for! 

Thanks :)

Reply:
I don't think we have this section in vNEXT.

Reply:
That looks to be true.

Reply:
https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2990